editor/src/components/canvas/canvas-strategies/strategies/absolute-resize.test-utils.tsx

```diff
@@ -5,6 +5,14 @@ import type { EdgePosition } from '../../canvas-types'
 import { mouseDragFromPointWithDelta } from '../../event-helpers.test-utils'
 import type { EditorRenderResult } from '../../ui-jsx.test-utils'
 
+export function getResizeControl(
+  renderResult: EditorRenderResult,
+  edgePosition: EdgePosition,
+): HTMLElement | null {
+  const testIdOfControl = `resize-control-${edgePosition.x}-${edgePosition.y}`
+  return renderResult.renderedDOM.queryByTestId(testIdOfControl)
```

Comment:
getByTestId automatically throws an error if it finds no elements or finds more than one element with the same test id

Reply:
This function is also being used in `resizeElement` for the case when it's not null.